api/view/static/openapi.json

```diff
@@ -0,0 +1,268 @@
+    }
+  ],
+  "paths": {
+    "/geometry": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "List geometries available",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/turtle": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/rdf+xml": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/ld+json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/n3": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/n-triples": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/geometry/{dataset_geomId}": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "Get geometry instance",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/turtle": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/geo+json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "getGeometryByDatasetIdGeomId",
+        "parameters": [
+          {
+            "name": "dataset_geomId",
+            "in": "path",
+            "description": "Dataset and Geometry ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instance",
```

Comment:
Search queries return result-sets in general. So isn't this 'Get geometry instances that contain specified position'

Reply:
copy and paste error ;)